Domain/Dto/BetableEntityDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Domain.Dto
+{
+    public class BetableEntityDto
+    {
+
+        [Required]
+        [StringLength(255)]
```

Comment:
I don't think we will need this on the dto.
I see there is no usage of this dto